src/foundry/foundry.js/avSettings.d.ts

```diff
@@ -1,37 +1,15 @@
 declare class AVSettings {
   constructor();
 
-  protected _set<T>(key: string, value: T): void;
-
-  protected _change(): void;
-
   client: AVSettings.ClientSettings;
 
   world: AVSettings.WorldSettings;
 
   protected _original: AVSettings.Settings;
```

Comment:
Are we really moving those around because the function containing them was called before the other two were defined? I don't think that's such a great idea. What about a function that is called in different places? I mean I get why make an exception here, but I would prefer to have it close to where the variables are defined first.

Reply:
Both ways are fine for me. I’ll move it back.

Reply:
done